Content.Server/Body/Systems/BloodstreamSystem.cs

```diff
@@ -128,13 +128,17 @@ public override void Update(float frameTime)
 
             // Removes blood from the bloodstream based on bleed amount (bleed rate)
             // as well as stop their bleeding to a certain extent.
-            if (bloodstream.BleedAmount > 0)
-            {
-                // Blood is removed from the bloodstream at a 1-1 rate with the bleed amount
-                TryModifyBloodLevel(uid, (-bloodstream.BleedAmount), bloodstream);
-                // Bleed rate is reduced by the bleed reduction amount in the bloodstream component.
-                TryModifyBleedAmount(uid, -bloodstream.BleedReductionAmount, bloodstream);
-            }
+            if (bloodstream.BleedAmount <= 0)
+                continue;
+
+            // Blood is removed from the bloodstream at a 1-1 rate with the bleed amount
+            TryModifyBloodLevel(uid, (-bloodstream.BleedAmount), bloodstream);
+
+            var modifyBleedLevel = new BleedStackReduceEvent(bloodstream.BleedReductionAmount);
```

Comment:
```suggestion
            var ev = new BleedStackReduceEvent(bloodstream.BleedReductionAmount);
```
usual naming convention